package-lock.json

```diff
@@ -1256,12 +1256,12 @@
       }
     },
     "node_modules/braces": {
-      "version": "3.0.2",
-      "resolved": "https://registry.npmjs.org/braces/-/braces-3.0.2.tgz",
-      "integrity": "sha512-b8um+L1RzM3WDSzvhm6gIz1yfTbBt6YTlcEKAvsmqCZZFw46z626lVj9j1yEPW33H5H+lBQpZMP1k8l+78Ha0A==",
+      "version": "3.0.3",
```

Comment:
IMO it doesn't make sense to just upgrade `braces` (transitive dep of jest), since we can upgrade `jest` and the rest of the dependencies shall follow